docs/guide.md

```diff
@@ -185,41 +192,48 @@ url-source:
 - cross
 ```
 
-You can add as many channels as you like. They are combined under *Last*, so versions from the prerelease channel
-here overwrite the default ones, if any.
+You can add as many release channels as you like. They are combined such that
+information in later channels takes precendence over information in earlier
+ones. In the example above, versions from the 'prerelease' channel overwrite
+the default ones, if any.
 
-To remove the channel, delete the entire `url-source` section or set it back to the default:
+To remove all release channels, delete the entire `url-source` section or set it
+to the default, namely:
 
 ```yml
 url-source:
   - GHCupURL
 ```
 
-Also see [config.yaml](https://github.com/haskell/ghcup-hs/blob/master/data/config.yaml)
+See [config.yaml](https://github.com/haskell/ghcup-hs/blob/master/data/config.yaml)
 for more options.
 
-You can also use an alternative metadata via the one-shot CLI option:
+You can also use an alternative metadata file via a one-shot option at the
+command line, as in the following example:
 
 ```sh
 ghcup --url-source=https://some-url/ghcup-0.0.9.yaml tui
```

Comment:
Here we can add this too

`ghcup --url-source=prereleases tui`

Reply:
Noted. Will add.